test/integration/sanSocialDominance.js

```diff
@@ -0,0 +1,59 @@
+const { testFieldTypes } = require('../helper.js')
+
+const {
+  testHistoricDataIsForbidden,
+  testHandlesNullData,
+  assertNumberOfRecords
+} = require('../integration_helper.js')
+
+const {
+  ethereumSlug,
+  from,
+  to,
+  source,
+  historicDataFrom,
+  historicDataTo,
+  days,
+  numberOfDays,
+  formatDate
+} = require('../setup.js')
+
+describe('SAN_SOCIAL_DOMINANCE', () => {
+  const expected = {
+    date: 'string',
+    dominance: 'number'
+  }
+
+  const response = san.SAN_SOCIAL_DOMINANCE(ethereumSlug, from, to, source)
+  const headers = response[0]
+  const results = response[1]
+  console.log(response)
```

Comment:
leftover

Reply:
Thanks, I've removed the leftover, the identation and the single row you've requested, also fixed the date